language/move-vm/runtime/src/logging.rs

```diff
@@ -27,3 +29,35 @@ impl NoContextLog {
 impl LogContext for NoContextLog {
     fn alert(&self) {}
 }
+
+//
+// Utility functions
+//
+
+pub fn expect_no_verification_errors(err: VMError, log_context: &impl LogContext) -> VMError {
```

Comment:
I found it a little bit weird to put this function here... How about moving it into the adapter?

Reply:
It is used by the loader too, not sure where else to put it